style.css

```diff
@@ -503,3 +513,247 @@ a {
 }
 
 /* About section End  */
+
+/* Desktop version Start  */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+    height: auto;
+  }
+
+  .menu-icon {
+    color: #6070ff;
+    display: none;
+  }
+
+  .menu-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-right: 20px;
+  }
+
+  .header {
+    width: 100%;
+    background: #fff;
+    height: 732px;
+    display: flex;
+    flex-direction: column;
+    background-image: url('images/headerbgdisktop.svg');
+    background-size: 75%;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
```

Comment:
- [ ] Great job so far 👏 👏  Your design looks good 🔥. In addition to what you have done, It would be better to add more spices to your design 😉. To be specific, **please add the curved end on the bottom-left of this header** section which is also the headline. Please try to look at the comparison below 👇

**Figma design**

![image](https://user-images.githubusercontent.com/90222110/186996621-c2fb01aa-4757-4b27-afce-ba81d993551d.png)

**Your design**

![image](https://user-images.githubusercontent.com/90222110/186996726-e9d627ee-500c-4c82-9593-1595d2e890ad.png)

As a hint, you can add the `border-bottom-left-radius` property on the header section but in order for it to appear you will need to set the `background-color` of the body to the grey color (i.e: #E5E5E5) 😉 😎. Another way is by using the clipping-path property and the best useful tool for that is [Clippy](https://bennettfeely.com/clippy/)

Please put much effort into implementing this design as it will enhance the overall design and also make your **portfolio unique** 🏃